clients/rust/examples/sysadmin-example.rs

```diff
@@ -1,12 +1,12 @@
 extern crate sysadmin_bindings;
 extern crate protobuf;
-extern crate error_chain;
+extern crate failure;
+#[macro_use]
+extern crate log;
 
-// note the wildcard import is recommended for error_chain
-// see https://docs.rs/error-chain
-use sysadmin_bindings::error_chain_generated_errors::*;
+use sysadmin_bindings::errors::SysadminResult;
 use sysadmin_bindings::{SysadminClient, Set, Get, Commit, Drop, CommitConfig, DumpHooks};
-
+type Result<T> = SysadminResult<T>;
```

Comment:
For such a small example, you really don't need this. If you did want to use failure here, the right way to go about it is to define everything all over again and not re-use sysadmin's error type.